### PR TITLE
fix: docker-compose.yml for delivery

### PR DESCRIPTION
**Pull request by disufi**
This is based on https://github.com/upfrontIO/livingdocs-bluewin-delivery/pull/134

### CODE REVIEW

File: delivery/docker-compose.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@ delivery:
 varnish:
   image: livingdocs/bluewin-varnish:latest
   environment:
-    BACKEND_HOST: localhost
+    BACKEND_HOST: delivery
     BACKEND_PORT: 9999
   ports:
-    - 8080:8080
+    - 80:80
```
Comment (marcbachmann) [inline delivery/docker-compose.yml]: https://github.com/upfrontIO/livingdocs-docker/pull/69/files#diff-f4d2792b7f6e01e4ce136f98b82bec9fR2

Comment (marcbachmann) [inline delivery/docker-compose.yml]: oh, we're in the docker repo. wrong alarm 😄

Comment (marcbachmann) [inline delivery/docker-compose.yml]: Do we really want to use port 80? That mans we'll have to start docker-compose with sudo.
I don't want to do that every time.

Comment (disufi) [inline delivery/docker-compose.yml]: Idk I only did the needed changes to make it work again.. the current bluewin delivery does not work at all when using it with varnish. I can make the relevant changes if anyone comes up with a better port to use (maybe `8080` as before?).

Comment (marcbachmann) [inline delivery/docker-compose.yml]: Oh, in that case `8080:80` would be better.

**Pull request merged**